ansible_risk_insight/finder.py

```diff
@@ -33,6 +36,10 @@
 from .safe_glob import safe_glob
 from .awx_utils import could_be_playbook, search_playbooks
 
+logging.basicConfig(
+    level=os.environ.get('LOGLEVEL', 'WARNING').upper()
+)
+
```

Comment:
ARI already has its own logger at `ansible_risk_insight.logger` and we can use it like [this](https://github.com/ansible/ansible-risk-insight/blob/main/ansible_risk_insight/scanner.py#L50), so please consider using it instead of the native logging library.
If the native logging is better here, we can accept this change, but at least the environment variable should be consistent with the existing one `ARI_LOG_LEVEL` (ref: [link](https://github.com/ansible/ansible-risk-insight/blob/main/ansible_risk_insight/scanner.py#L117))